src/uct/ib/dc/dc_mlx5.h

```diff
@@ -40,8 +41,8 @@ struct ibv_ravh {
 #define UCT_DC_MLX5_IFACE_TXQP_DCI_GET(_iface, _dci, _txqp, _txwq) \
     { \
         ucs_assert(_dci != UCT_DC_MLX5_EP_NO_DCI); \
-        _txqp = &(_iface)->tx.dcis[_dci].txqp; \
-        _txwq = &(_iface)->tx.dcis[_dci].txwq; \
+        _txqp = &ucs_array_elem(&(_iface)->tx.dcis, _dci).txqp; \
```

Comment:
maybe add a helper function : `uct_dc_mlx5_iface_dci(iface, dci_index)` to return pointer to a given dci by index
seems it can be used in many places